marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/cloud/CloudProviderUtil.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2012-2020 MarkLogic Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.marklogic.hub.central.cloud;
```

Comment:
I gave bad advice on this class package. It doesn't have anything to do with cloud providers, but it is web-specific and SSL-specific. So let's call this SslUtil and toss it com.marklogic.hub.central.web (and at some point, we probably ought to move our controllers there, but that's a separate matter).